spec/factories.rb

```diff
@@ -385,28 +386,28 @@ def r_str
   factory :report do
     user
     association :item, factory: :status_message
-    text "offensive content"
+    text { "offensive content" }
   end
 
   factory :o_auth_application, class: Api::OpenidConnect::OAuthApplication do
     client_name { "Diaspora Test Client #{r_str}" }
-    redirect_uris %w(http://localhost:3000/)
+    redirect_uris { %w(http://localhost:3000/) }
```

Comment:
Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`.